bundles/org.openhab.binding.semsportal/README.md

```diff
@@ -0,0 +1,45 @@
+
+## Supported Things
+
+There is only one thing type supported in this binding, which is the powerStation. You need to provide the identifying UUID of the solar system in your account yourself. If you have multiple powerstations that report to the same account, you can configure the same account for each powerSation and only differ the unique id of the station in each Thing.
+
+## Discovery
+
+At this moment there is no discovery possible. Configuration of the powerStation unique id is done by hand in the Thing configuration page.
+
+## Thing Configuration
+
+The configuration of the thing is pretty straight forward. You need to have your power station set up in the SEMS portal, and you need to have an account that is allowed to view the power station data. You should log in at least once in the portal with this account to activate it. 
+
+The thing needs the username and password to connect and retreive the data. It also needs the unique id of the power station. It can be found after you log in to the portal, on the URL. use the value you find in the URL at the x-es (including the -s) in this example: https://www.semsportal.com/powerstation/powerstatussnmin/xxxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+
+
+## Channels
+
+
+| channel       | type      | description                                                                                                |
+| ------------- | --------- | ---------------------------------------------------------------------------------------------------------- |
+| onlineState   | Switch    | If the powerStation is currently on and reporting to the portal                                            |
+| lastUpdate    | DateTime  | Last time the powerStation sent information to the portal                                                  |
+| currentOutput | Number    | The current output of the powerStation in Watt                                                             |
+| todayTotal    | Number    | Todays total generation of the station in kWh                                                              |
+| monthTotal    | Number    | This month's total generation of the station in kWh                                                        |
+| overallTotal  | Number    | The total generation of the station since installation, in kWh                                             |
+| todayIncome   | Number    | Todays income as reported by the portal, if you have configured the power rates of your energy provider    |
+| totalIncome   | Number    | The total income as reported by the portal, if you have configured the power rates of your energy provider |
```

Comment:
Can you implement these with Units Of Measure, aka UOM?
You get automatic graphs drawn by OH3 if you implement it this way.
If you need a similar binding with examples to follow check out this one that is also getting reviewed that is found here:
https://github.com/Hilbrand/openhab-addons/blob/enphase/bundles/org.openhab.binding.enphase/src/main/resources/OH-INF/thing/thing-types.xml

Reply:
Done